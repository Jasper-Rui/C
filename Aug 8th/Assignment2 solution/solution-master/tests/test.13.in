TRX 11235 Alice_Q_Public Bob_X_Private 150000 20
TRX 12358 Carol_Y_Incredible Dave_Z_Particle 350000 40
TRX 23581 Eve_N_Star Fred_F_Filibuster 1350000 10
TRX 35813 Gina_T_Positron Hugh_H_Haliboo 11350000 100
TRX 58132 Bob_X_Private Alice_Q_Public 150000 20
MINE
MINE
END

