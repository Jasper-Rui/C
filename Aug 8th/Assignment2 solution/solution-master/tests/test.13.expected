Adding transaction: 11235 Alice_Q_Public Bob_X_Private 150000 20
Adding transaction: 12358 Carol_Y_Incredible Dave_Z_Particle 350000 40
Adding transaction: 23581 Eve_N_Star Fred_F_Filibuster 1350000 10
Adding transaction: 35813 Gina_T_Positron Hugh_H_Haliboo 11350000 100
Adding transaction: 58132 Bob_X_Private Alice_Q_Public 150000 20
Block mined: 1 0 0x00000000 5
11235 Alice_Q_Public Bob_X_Private 150000 20
12358 Carol_Y_Incredible Dave_Z_Particle 350000 40
23581 Eve_N_Star Fred_F_Filibuster 1350000 10
35813 Gina_T_Positron Hugh_H_Haliboo 11350000 100
58132 Bob_X_Private Alice_Q_Public 150000 20
0x00000101 0x006aa499
Block mined: 2 1 0x006aa499 0
0x00000101 0x004f0056
